src/components/application_manager/include/application_manager/application.h

```diff
@@ -880,17 +893,11 @@ class Application : public virtual InitialApplicationData,
    */
   virtual bool RemoveExtension(AppExtensionUID uid) = 0;
 
-  /**
-   * @brief Removes all extensions
-   */
-  virtual void RemoveExtensions() = 0;
-
   /**
    * @brief Get list of subscriptions to vehicle info notifications
    * @return list of subscriptions to vehicle info notifications
    */
   virtual const VehicleInfoSubscriptions& SubscribesIVI() const = 0;
```

Comment:
why not move SubscribesIVI to vehicle info plugin?

Reply:
@yang1070  all functions regarding subscriptions are already moved to vehicle_info plugin, please look at the latest updates